superset-frontend/src/messageToasts/components/Toast.tsx

```diff
@@ -97,13 +117,17 @@ class Toast extends React.Component {
           toastType === DANGER_TOAST && 'toast--danger',
         )}
       >
-        <Interweave content={text} />
+        <ToastContianer>
+          {toastType === SUCCESS_TOAST ? (
+            <Icon name="check" />
+          ) : (
+            <Icon name="error" />
+          )}
```

Comment:
Since we have a warning icon, should we use that for the `WARNING_TOAST`s too? And perhaps add an info icon as well? If we don't add the info icon, should we perhaps default to no icon at all, since it could be confusing for an info toast to show an error icon

Reply:
I will prefer info toasts without Icon and warning toasts will have the same icon as error.